modules/microshift-install-rpm-before.adoc

```diff
@@ -6,6 +6,14 @@
 [id="microshift-install-rpm-before_{context}"]
 = Before installing {microshift-short} from an RPM package
 
+Preparation of the host machine is recommended prior to installing {microshift-short} for memory configuration and FIPS compliance.
+
+== Configuring volume groups
+
 {microshift-short} uses the logical volume manager storage (LVMS) Container Storage Interface (CSI) plugin for providing storage to persistent volumes (PVs). LVMS relies on the Linux logical volume manager (LVM) to dynamically manage the backing logical volumes (LVs) for PVs. For this reason, your machine must have an LVM volume group (VG) with unused space in which LVMS can create the LVs for your workload's PVs.
 
 To configure a volume group (VG) that allows LVMS to create the LVs for your workload's PVs, lower the *Desired Size* of your root volume during the installation of {op-system}. Lowering the size of your root volume allows unallocated space on the disk for additional LVs created by LVMS at runtime.
+
+== Running containers in FIPS mode
+
+If your use case requires running {microshift-short} containers in FIPS mode, you must install {op-system-base} with FIPS enabled. After the worker machine is configured to run in FIPS mode, your {microshift-short} containers are automatically configured to also run in FIPS mode. See "Enabling FIPS compliance with RPM-based installations" in the Additional resources.
```

Comment:
Clarify the "Additional resources" mention / match similar style elsewhere in repo:
```suggestion
If your use case requires running {microshift-short} containers in FIPS mode, you must install {op-system-base} with FIPS enabled. After the worker machine is configured to run in FIPS mode, your {microshift-short} containers are automatically configured to also run in FIPS mode. See "Enabling FIPS compliance with RPM-based installations" in the "Additional resources" of this section.
```
Although the "Additional resources" of this section actually links to "Running containers in FIPS mode", so I'm not sure which is correct. I'm guessing the following?
```suggestion
If your use case requires running {microshift-short} containers in FIPS mode, you must install {op-system-base} with FIPS enabled. After the worker machine is configured to run in FIPS mode, your {microshift-short} containers are automatically configured to also run in FIPS mode. See "Running containers in FIPS mode" in the "Additional resources" of this section.
```
Otherwise, adjust the `xref` at the assembly level to refer to "Enabling FIPS compliance with RPM-based installations".